components/SpotIM/index.js

```diff
@@ -35,6 +35,7 @@ class SpotIM extends Component {
       } else {
         this.onApiReady = this.sso;
         document.addEventListener('spot-im-api-ready', this.onApiReady, false);
+        document.addEventListener('spot-im-login-start', this.onApiReady);
```

Comment:
missing the (annoying) last argument
```suggestion
        document.addEventListener('spot-im-login-start', this.onApiReady, false);
```